src/main/java/org/jabref/logic/formatter/bibtexfields/NormalizePagesFormatter.java

```diff
@@ -63,7 +64,8 @@ public String format(String value) {
         // Remove pages prefix
         String cleanValue = value.replace("pp.", "").replace("p.", "");
         // remove unwanted literals including en dash, em dash, and whitespace
-        cleanValue = cleanValue.replaceAll("\u2013|\u2014", "-").replaceAll(REJECT_LITERALS, "");
+        cleanValue = EM_EN_DASH_PATTERN.matcher(cleanValue).replaceAll("-");
+        cleanValue = REJECT_LITERALS_PATTERN.matcher(cleanValue).replaceAll("");
```

Comment:
I think this may result in unwanted changes. The formatter should only format pages if they are "kind of" in the right format. Now something like `40&50` is changed to `4050`.

Reply:
Fixed in the new commit